### ISSUE TITLE
jsonnetfmt not available in 0.13

### ISSUE THREAD

**Issue by bpedman** [bitnami/bitnami-docker-jsonnet#5]
In jsonnet 0.13 the `fmt` subcommand was moved to its own executable `jsonnetfmt`, see https://github.com/google/jsonnet/commit/08875537f6e350a8d2acbd548bdd997b80cd86c9

This executable is not available in the docker container unfortunately. Could it get added to the container? 

While its possible to just dump the executable in there I wonder if the entrypoint could also be made a little more dynamic to specify docker run arguments that would trigger running jsonnetfmt rather than just having the entrypoint set directly to jsonnet. It's fairly easy to override with `--entrypoint jsonnetfmt` but its not natural, I had to look that up. But thats just to make it nicer, its fine if it was just available and could use `--entrypoint jsonnetfmt`

**Comment by juan131** [comment]
Hi @bpedman 

Thanks so much for sharing your feedback! As you pointed out, we're missing that binary in the image. I just created a task to include it.

I'd let you know as soon as we release a new image including it! Thanks again for reporting it!

**Comment by juan131** [comment]
Hi @bpedman 

We just released a new revision (**0.13.0-r2**) of the image that contains both binaries:

```bash
$ docker run -it --entrypoint /bin/bash bitnami/jsonnet:0.13.0-r2
# which jsonnet
/opt/bitnami/jsonnet/bin/jsonnet
# which jsonnetfmt
/opt/bitnami/jsonnet/bin/jsonnetfmt
```

I'm closing this issue as solved, please feel free to reopen it if you require further assistance.

**Comment by bpedman** [comment]
Thank you for the quick turnaround, it works :)
